### ISSUE TITLE
Remove android-api level 16 test for Fennec on mozilla-central

### ISSUE THREAD

**Issue by whimboo** [mozilla/mozdownload#565]
Currently we experience failures in tests because the Android API level 16 Nightly builds of Fennec for mozilla-central are no longer existent for download. This is due to the deprecation plan.

https://archive.mozilla.org/pub/mobile/nightly/

As such the tests have to be removed here:

https://github.com/mozilla/mozdownload/blob/master/tests/remote/test_fennec.py

**Comment by whimboo** [comment]
@claudius96 would you be interested to work on this issue?

**Comment by whimboo** [comment]
Actually this also includes the x86 versions of Fennec.

**Comment by Divyanshu29** [comment]
If @claudius96 isn't interested to work on this issue I am ready to work on this issue! Just waiting for him to confirm his decision!

**Comment by whimboo** [comment]
@Divyanshu29 I think feel free to get started! If @claudius96 has interest I can find something else. Important for now is to get you unblocked for the other PR. Thanks.

**Comment by Divyanshu29** [comment]
@whimboo Can u pls have a look at PR now! I think it shd be working fine now... I tested it with tox and main issue was from #565 and some errors not related to part of code which i changed!

**Comment by whimboo** [comment]
@Divyanshu29 please provide a separate PR for this particular issue which needs to be fixed first. Thanks.

**Comment by Divyanshu29** [comment]
I was referring to my previous PR #564 related to issue #445

**Comment by whimboo** [comment]
This was fixed by PR #566. Thanks @divyanshu29 for working on it!
